src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue

```diff
@@ -0,0 +1,339 @@
+<!--
+   The MIT License
+   Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+   Copyright (c) 2018 Estonian Information System Authority (RIA),
+   Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+   Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to deal
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+ -->
+<template>
+  <sub-view-container>
+    <!-- Title  -->
+    <div class="table-toolbar align-fix mt-0 pl-0">
+      <div class="xrd-view-title align-fix">
+        {{ $t('tab.settings.systemSettings') }}
+      </div>
+    </div>
+
+    <div>
+      <!-- System Parameters -->
+
+      <div id="anchor" class="mb-6">
+        <v-card class="pb-4" flat>
+          <div class="card-top">
+            <div class="card-main-title">
+              {{ $t('systemSettings.systemParameters') }}
+            </div>
+          </div>
+
+          <table class="xrd-table mt-0 pb-3">
+            <tbody>
+              <tr>
+                <td class="title-cell">
+                  <div>
+                    <div>
+                      {{ $t('systemSettings.instanceIdentifier') }}
+                    </div>
+                  </div>
+                </td>
+                <td>DEV</td>
+                <td></td>
+              </tr>
+              <tr>
+                <td>
+                  <div>
+                    <div>
+                      {{ $t('systemSettings.centralServerAddress') }}
+                    </div>
+                  </div>
+                </td>
+                <td>DEV</td>
```

Comment:
Maybe make all the tables dynamic already in the mock. It should give more value to the mocks and makes it quicker to hook them up with backend later on.

Reply:
Fixed